BaseTools/Source/C/GenStm/Makefile

```diff
@@ -0,0 +1,17 @@
+## @file
+# Windows makefile for the GenStm C tool build.
+#
+# Copyright (c) Microsoft Corporation

```

Comment:
Is this really Microsoft Corp copyright?

Reply:
It previously had no copyright, so I assumed it originated in our changes. I'll trace the lineage.

Reply:
Updated to Intel copyright based on code in https://github.com/jyao1/STM/blob/master/Stm/StmPkg/EdkII/BaseTools/Source/C/GenStm/Makefile